Content/Serializer/MediaSerializer.php

```diff
@@ -80,21 +80,28 @@ public function serialize(MediaInterface $media, string $locale, ?SerializationC
         unset($mediaData['adminUrl']);
         unset($mediaData['_hash']);
 
+        $formatMediaApi = $apiMedia;
+        /** @var Media|null $previewImage */
```

Comment:
Sorry my fault this need to be as it is not an api media instead the media entity:

```suggestion
        /** @var MediaInterface|null $previewImage */
```